core/src/main/kotlin/fr/sncf/osrd/sim_infra_adapter/RawInfraFromRjsAdapter.kt

```diff
@@ -331,7 +429,13 @@ fun adaptRawInfra(infra: SignalingInfra, rjsInfra: RJSInfra): SimInfraAdapter {
         val trackSectionBlockedGauges = getBlockedGauge(rjsTrack)
 
         var chunkStartOffset = Offset<TrackSection>(0.meters)
-        val chunkEndOffsets = trackSectionNameToChunkEndOffsets[rjsTrack.id] ?: mutableSetOf()
+        val trackSectionDistanceSortedWaypointIdxs =
+            trackSectionNameToDistanceSortedWaypointIdxs[rjsTrack.id] ?: TreeMap()
+        // ignore 0 offset that is a chunk's start
```

Comment:
The comment is a little unclear, maybe something like this instead?

```suggestion
        // split chunks on any detector that isn't placed at offset=0
```

Reply:
Done: https://github.com/osrd-project/osrd/pull/6921/commits/754865f7833fe40281a8789e8f79108f55b6a2dc#diff-0335c3a24d409feef2e7c3f5f3173cc4d08928c658e17faa8a0e71384922e6d3R424